### DISCUSSION TITLE
Show new Followers/Hosts/Raids only in my chat

### DISCUSSION THREAD

**Discussion by Nerxxy** [Chatterino/chatterino2#3693] [Ideas]
If this feature were to be added it would mean that I would never need to keep obs up on my screen to see who followed, hosted or raided, and I could then have something else up entirely.

So say I have my own chat open in chatterino this would show up:

![Untitled-1 copy](https://user-images.githubusercontent.com/37718547/165001958-1fbcdf39-b6ec-404f-9d07-454cd4548e5b.png)

But if I'm in another persons channel it doesn't show up.

**Comment by LosFarmosCTL** [comment]
I don't understand this request completely... are you just asking for an option to show raids etc. only for your own channel? And if yes, what would you need that for, what's the problem with it showing up in every channel?

**Reply by Nerxxy** [reply]
That's correct yes.

I would like it so I can save space on my other monitor to use for something other then obs, on one screen I have Spotify and chatterino, another screen my game and the last for obs, discord or something else entirely, I personally would like to have discord or something else open more then obs and don't want to keep having to alt tab a lot.

And with it not showing in every channel, I personally just don't want/need to see who followed the person I'm watching via a message in the chat, since it already shows up on their stream and usually the streamer shouts them out, some streamers also have a bot to tell who followed/hosted, so thats already 2/3 ways of knowing who followed them, don't need a 3rd/4th.

The feature just makes it so I wont need obs open all the time, plus not having obs running in the foreground will save resources not that it takes up much anyway.

Its just a small thing that will help me and possibly some other people, I know a guy that also uses chatterino for his stream chat and I feel it might be beneficial to some people that use it this way.

**Comment by jupjohn** [comment]
I don't think this feature is necessary. You can see who follows/hosts by using a 3rd party bot (StreamElements, Fossabot, Nightbot etc.). Raids already show in chatterino. 

As for keeping it to your own chat, I believe this would be a good use for filters. Check out the chatterino wiki for more information on how to use them. 

If chatterino was notify on new followers/hosts you'd end up with a huge amount of spam in large chats - unless you created a filter for it for all channels but yours. No point adding something this verbose to chatterino when you're able to observe those events using a chat bot.

**Comment by Nerxxy** [comment]
Now that i think about it, instead of it showing up in the chat it could just be a tab, like how we have the /live tab that shows when select people go live, just have an "/events" type tab that shows the raids/hosts/followers/subs since we can have multiple things in one tab
